### ISSUE TITLE
Crashes on open with Android O

### ISSUE THREAD

**Issue by artemist** [keybase/client#7072]
Keybase for Android crashes on Android O beta 1 on startup.

It seems to be crashing due to secomp blocking syscall 142, which is select. Unfortunately, I don't know exactly where the failure happened, This could be an upstream issue in libgojni.

The log is as follows:
`4415 --------- beginning of crash
4416 05-20 08:47:12.737 12353 12373 F libc    : Fatal signal 31 (SIGSYS), code 1 in tid 12373 (ybase.ossifrage)
4417 05-20 08:47:12.911 12379 12379 I crash_dump32: obtaining output fd from tombstoned
4418 05-20 08:47:12.915  3555  3555 I /system/bin/tombstoned: received crash request for pid 12353
4419 05-20 08:47:12.918 12379 12379 I crash_dump32: performing dump of process 12353 (target tid = 12373)
4420 05-20 08:47:12.918 12379 12379 F DEBUG   : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
4421 05-20 08:47:12.918 12379 12379 F DEBUG   : Build fingerprint: 'google/bullhead/bullhead:O/OPP2.170420.019/4021594:     user/release-keys'
4422 05-20 08:47:12.918 12379 12379 F DEBUG   : Revision: 'rev_1.0'
4423 05-20 08:47:12.918 12379 12379 F DEBUG   : ABI: 'arm'
4424 05-20 08:47:12.918 12379 12379 F DEBUG   : pid: 12353, tid: 12373, name: ybase.ossifrage  >>> io.keybase.ossifrage      <<<
4425 05-20 08:47:12.918 12379 12379 F DEBUG   : signal 31 (SIGSYS), code 1 (SYS_SECCOMP), fault addr --------
4426 05-20 08:47:12.918 12379 12379 F DEBUG   : Cause: seccomp prevented call to disallowed arm system call 142
4427 05-20 08:47:12.918 12379 12379 F DEBUG   :     r0 00000000  r1 00000000  r2 00000000  r3 00000000
4428 05-20 08:47:12.919 12379 12379 F DEBUG   :     r4 ccbe4890  r5 00000000  r6 00000000  r7 0000008e
4429 05-20 08:47:12.919 12379 12379 F DEBUG   :     r8 00003041  r9 ddbacfb0  sl 8aac85a0  fp 00002710
4430 05-20 08:47:12.919 12379 12379 F DEBUG   :     ip eb49463c  sp ccbe488c  lr ccf918a8  pc ccf918cc  cpsr 800f0010
4431 05-20 08:47:12.972 12379 12379 F DEBUG   :
4432 05-20 08:47:12.972 12379 12379 F DEBUG   : backtrace:
4433 05-20 08:47:12.972 12379 12379 F DEBUG   :     #00 pc 002af8cc  /data/app/io.keybase.ossifrage-YXGNVz8AE6HSY_XlWr-     u-A==/lib/arm/libgojni.so (runtime.usleep+48)
4434 05-20 08:47:12.972 12379 12379 F DEBUG   :     #01 pc 00000000  <unknown>
4435 05-20 08:47:13.545  3887 12380 W ActivityManager:   Force finishing activity io.keybase.ossifrage/.MainActivity
`

**Comment by cjb** [comment]
Yep, there's a fix in progress, see https://github.com/golang/go/issues/20409.

**Comment by NoahSchmitz** [comment]
Having the same issue on my Nexus 6P with O. Glad to here there is a fix in the works!

**Comment by cbushong-fearless** [comment]
Looks like golang/go#20409 has been resolved.  Any idea when things will get fixed in Android O?

**Comment by jfinkhaeuser** [comment]
#7672
